.dagger/src/GotenbergBundle.php

```diff
@@ -0,0 +1,242 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\DaggerFunction;
+use Dagger\Attribute\DaggerObject;
+use Dagger\Attribute\DefaultPath;
+use Dagger\Attribute\Doc;
+use Dagger\Attribute\ReturnsListOfType;
+use Dagger\Container;
+use Dagger\Directory;
+use Dagger\Service;
+use function Dagger\dag;
+
+#[DaggerObject]
+#[Doc('Module for GotenbergBundle')]
+class GotenbergBundle
+{
+    #[DaggerFunction]
+    #[Doc('Returns a Gotenberg container.')]
+    public function gotenbergContainer(
+        string $gotenbergVersion = '8.0',
+    ): Container {
+        return dag()
+            ->container()
+            ->from("gotenberg/gotenberg:{$gotenbergVersion}")
+        ;
+    }
+
+    #[DaggerFunction]
+    #[Doc('Returns a Gotenberg service.')]
+    public function gotenbergService(
+        string $gotenbergVersion = '8.0',
+    ): Service {
+        return $this->gotenbergContainer($gotenbergVersion)
+            ->withExposedPort(3000)
+            ->asService()
+        ;
+    }
+
+    #[DaggerFunction]
+    #[Doc('Returns a PHP container.')]
+    public function phpContainer(
+        #[DefaultPath('.')]
+        Directory $source,
+
+        string $phpVersion = '8.4',
+    ): Container {
+        $aptCache = dag()->cacheVolume("apt-cache-{$phpVersion}");
+        $composerBin = dag()->container()->from('composer/composer')->file('/usr/bin/composer');
+
+        return dag()
+            ->container()
+            ->from("php:{$phpVersion}")
+            ->withMountedCache('/var/cache/apt/archives', $aptCache)
+            ->withExec(['apt', 'update'])
+            ->withExec(['apt', 'install', '--yes',
+                'git',
+                'zip',
+            ])
+            ->withMountedFile('/usr/bin/composer', $composerBin)
+            ->withEnvVariable('COMPOSER_ALLOW_SUPERUSER', '1')
+            ->withWorkdir('/GotenbergBundle')
+            ->withMountedDirectory('/GotenbergBundle', $source)
+        ;
+    }
+
+    #[DaggerFunction]
+    #[Doc('Returns a PHP container with symfony set to the desired version.')]
+    public function symfonyContainer(
+        #[DefaultPath('.')]
+        Directory $source,
+
+        string $phpVersion = '8.4',
+        string $symfonyVersion = '7.3',
+        Container|null $phpContainer = null,
+    ): Container {
```

Comment:
```suggestion
    private function symfonyContainer(
        Directory $source,
        string $phpVersion = '8.4',
        string $symfonyVersion = '7.3',
        Container|null $phpContainer = null,
    ): Container {
```

Similar to the `phpContainer` function, if this is an internal method, no need to expose it to Dagger.